plugins/experimental/traffic_dump/traffic_dump.cc

```diff
@@ -278,6 +290,27 @@ collect_headers(TSMBuffer &buffer, TSMLoc &hdr_loc, int64_t body_bytes)
   return result + "]}}";
 }
 
+/// Helper functions to collect txn information from TSMBuffer
+std::string
+collect_headers(TSMBuffer &buffer, TSMLoc &hdr_loc, std::string_view body)
+{
+  std::string content_field;
+  content_field += R"(,"content":{"encoding":"esc_json","size":)" + std::to_string(body.length());
+  if (!body.empty()) {
+    content_field += R"(,"data":")" + escape_json(std::string(body)) + R"(")";
+  }
+  content_field += '}';
```

Comment:
Doesn't performance matter here, so maybe use FixedBufferWriter and alloca() to get rid of some heap allocation calls?